src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -44,14 +48,11 @@
     margin: -($border-width + 2px);
   }
 
-  .mat-focus-indicator.mat-stroked-button::before {
+  .mat-focus-indicator.mat-stroked-button::before,
+  .mat-focus-indicator.mat-calendar-body-cell-content::before {
```

Comment:
This is a bug fix caused by https://github.com/angular/components/pull/19304. Given we moved the focus indicator to a different element, the offset it requires to have sufficient contrast now changes. We actually get to net remove lines of code here, which is nice.